EC-Delete-LandingZone.py

```diff
@@ -0,0 +1,529 @@
+#!/usr/bin/python -u
+import signal
+import sys, getopt
+import subprocess, pkg_resources
+import os
+import subprocess
+import shlex
+import logging
+import time
+import boto3
+import boto3.session
+import botocore
+import json
+import threading
+import cursor
+import yaml
+
+from datetime import datetime
+from enum import Enum
+from colorama import Fore, Back, Style
+from botocore.exceptions import BotoCoreError, ClientError, ProfileNotFound
+from botocore.config import Config
+
+regions = ["ap-northeast-1","ap-northeast-2","ap-northeast-3","ap-south-1","ap-southeast-1","ap-southeast-2","ca-central-1","eu-central-1","eu-north-1","eu-west-2","eu-west-3","sa-east-1","us-east-1","us-east-2","us-west-1","us-west-2"]
+
+def main(argv):
+    
+    verbosity = logging.ERROR
+    start_time = time.time()
+  
+    seclog_profile = None
+    seclog_id = None
+
+    account_profile = None
+    account_id = None
+    stored_seclog_id = None
+
+    try:
+        opts, args = getopt.getopt(argv,"hva:s:",["account", "seclog", "verbose"])
+    except getopt.GetoptError:
+        usage()
+        sys.exit(2)
+    
+    print("#######")
+    print("####### AWS Landing Zone delete script")
+    print("")
+
+
+    # Parsing script parameters
+    for opt, arg in opts:
+        if opt == '-h':
+            usage()
+            sys.exit()
+        if opt in ("-v", "--verbose"):
+            verbosity = logging.DEBUG
+        if opt in ("-a", "--account"):
+            if not arg:
+                print(f"Account profile has not been provided. [{Status.FAIL.value}]")
+                usage()
+                sys.exit(1)
+            else:
+                account_profile = arg
+        
+        if opt in ("-s", "--seclog"):
+            if not arg:
+                print(f"Source SECLOG profile has not been provided. [{Status.FAIL.value}]")
+                usage()
+                sys.exit(1)
+            else:
+                seclog_profile = arg
+        
+    
+    logging.basicConfig(level=verbosity)
+
+    # linked account
+    
+    print(f"Account to be processed : {account_profile}")
+    account_session=boto3.Session(profile_name=account_profile, region_name ='eu-west-1')
+    
+    account_id = get_account_id(account_session)
+
+    print(f"Account ID : {account_id}")
+    
+    # get stored seclog_id from ssm parameter
+    ssm = account_session.client('ssm')
+    try:
+        response = ssm.get_parameter(Name='/org/member/SecLogMasterAccountId')
+        stored_seclog_id = response['Parameter']['Value']
+        if is_seclog(account_session) == True:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The `==` and `!=` operators use the compared objects&#39; ` __eq__` method to test if they are equal. To check if an object is a singleton, such as `None`, we recommend that you use the `is` identity comparison operator.

[Learn more](https://docs.python.org/3/reference/expressions.html#is)

Similar issue at line number 104.